nextcord/ui/view.py

```diff
@@ -177,9 +184,16 @@ def __init_subclass__(cls) -> None:
 
         cls.__view_children_items__ = children
 
-    def __init__(self, *, timeout: Optional[float] = 180.0, auto_defer: bool = True):
+    def __init__(
+        self,
+        *,
+        timeout: Optional[float] = 180.0,
+        auto_defer: bool = True,
+        prevent_update: bool = False,
+    ):
         self.timeout = timeout
         self.auto_defer = auto_defer
+        self.prevent_update = prevent_update if not timeout else True
```

Comment:
By default this will always be enabled? I thought it was going to be disabled by default for backwards compatibility

Reply:
This feat is supposed to be a bug fix as well. In most if not all cases, you don't want a persistent view to be individually stored for each message, as the view is persistent for a reason: so you don't have to create a view for each message, and that it works across reboots. This feat should fix memory leaks caused by views unknowingly being stored in ViewStore and not being deleted because the view timeout is `None`. By default, this being enabled *should* be backwards compatible to anyone correctly storing their persistent views with `bot.add_view`, and shouldn't affect those using non-persistent views.

Reply:
This is not backwards compatible though. Since they may use attributes on the view for the first lifecycle. You may say "they shouldn't do that", but it is still possible and is used. This change in behaviour is a breaking change and should be opt-in.

Reply:
I've changed it to default to opt-in. 
But that kind of defeats the original purpose of this PR if they don't enable it. Why would a breaking change requiring a small amount of people to fix their code be bad in order to fix a memory leak issue, arguably a larger issue

Reply:
Because it isn't a "small amount of people". You cannot base how many users use a feature based off your own experience, since the majority of nextcord users are not even in the Discord server.